msu/systems/registry/registry_mod_addon.nut

```diff
@@ -0,0 +1,63 @@
+::MSU.Class.RegistryModAddon <- class extends ::MSU.Class.SystemModAddon
+{
+	static GithubURLRegex = regexp("https:\\/\\/github\\.com\\/([-\\w]+)\\/([-\\w]+)");
+	static NexusURLRegex = regexp("https:\\/\\/www\\.nexusmods\\.com\\/battlebrothers\\/mods\\/(\\d+)");
+	__GithubURL = null;
+	__NexusModsURL = null;
+	__UpdateSource = null;
+
+	function getGithubURL()
+	{
+		return this.__GithubURL;
+	}
+
+	function setGithubURL( _url )
+	{
+		::MSU.requireString(_url);
+		if (!this.GithubURLRegex.match(_url))
+		{
+			::logError("A Github link must be a link to a specific repository, eg: https://github.com/MSUTeam/MSU");
```

Comment:
Probably add something like "Is there an issue with your URL"?

Reply:
Sure